tutorials/basics/stream/stream2.py

```diff
@@ -20,14 +20,6 @@ class StreamWithRefTest(rfm.RegressionTest):
         'OMP_NUM_THREADS': '4',
         'OMP_PLACES': 'cores'
     }
-    reference = {
-        'catalina': {
-            'Copy':  (25200, -0.05, 0.05, 'MB/s'),
-            'Scale': (16800, -0.05, 0.05, 'MB/s'),
-            'Add':   (18500, -0.05, 0.05, 'MB/s'),
-            'Triad': (18800, -0.05, 0.05, 'MB/s')
-        }
-    }
```

Comment:
Since we removed the reference here, we should probably rename the test as well.